incubator/hnc/api/v1alpha1/hierarchical_namespace.go

```diff
@@ -43,6 +44,10 @@ type HierarchicalNamespaceStatus struct {
 	//
 	// - "conflict": a namespace of the same name already exists. If the validation webhook
 	// works properly, this should never happen since the HNS won't be created at the first place.
+	//
+	// - "error": there's something wrong with the HNS. Currently, it would be that
+	// the parent namespace is not allowed to have subnamespaces. If the validation webhook
+	// works properly, this should not happen since the HNS won't be created at the first place.
```

Comment:
Maybe instead of "If the validation webhook works properly...", replace this with "The admission controller will attempt to prevent this."?